ports/servoshell/desktop/minibrowser.rs

```diff
@@ -175,36 +183,39 @@ impl Minibrowser {
             // TODO: While in fullscreen add some way to mitigate the increased phishing risk
             // when not displaying the URL bar: https://github.com/servo/servo/issues/32443
             if window.fullscreen().is_none() {
-                TopBottomPanel::top("toolbar").show(ctx, |ui| {
+                let frame = egui::Frame::default()
+                    .fill(Color32::from_gray(32))
+                    .inner_margin(4.0);
+                TopBottomPanel::top("toolbar").frame(frame).show(ctx, |ui| {
                     ui.allocate_ui_with_layout(
                         ui.available_size(),
                         egui::Layout::left_to_right(egui::Align::Center),
                         |ui| {
-                            if ui.button("back").clicked() {
+                            if ui.add(Minibrowser::round_button("⏴")).clicked() {
                                 event_queue.borrow_mut().push(MinibrowserEvent::Back);
                             }
-                            if ui.button("forward").clicked() {
+                            if ui.add(Minibrowser::round_button("⏵")).clicked() {
                                 event_queue.borrow_mut().push(MinibrowserEvent::Forward);
                             }
+
+                            match self.load_status {
+                                LoadStatus::LoadStart | LoadStatus::HeadParsed => {
+                                    if ui.add(Minibrowser::round_button("X")).clicked() {
+                                        event_queue.borrow_mut().push(MinibrowserEvent::Reload);
```

Comment:
Yeah, but currently there seems to be no `EmbedderEvent` for that and I wanted to keep this PR focused on the minibrowser.

Reply:
Alright, I've amended the commit, now we'll log a message using `warn!` (I hope that's the right way to log here).

I've also renamed the `rounded_button` function to `toolbar_button` since they're not particularly round anymore.